www-cronrobot/app/javascript/stylesheets/application.scss

```diff
@@ -1,5 +1,534 @@
-@import './bootswatch-variables.scss';
-@import './bootstrap.css';
-@import './bootswatch-core.scss';
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800;900&display=swap');
 
-@import './font-awesome.scss';
\ No newline at end of file
+html {
+    scroll-behavior: smooth;
+}
+
+body {
+    background: black;
+    font-family: 'Montserrat', sans-serif;
+    margin: 0 auto;
+    transition: .3s linear;
+    overflow-x: hidden;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:active {
+    color: inherit;
+}
+
+/* General Home Styling */
+
+#home {
+    width: 100vw;
+    height: calc(550px + 20vw);
+    display: flex;
+    align-items: center;
+    /* justify-content: center; */
+    flex-direction: column;
+}
+
+#heading {
+    color: white;
+    margin-top: calc(180px + 1.5vw);
+    font-size: calc(25px + 1.5vw);
+    text-align: center;
+    max-width: calc(400px + 25vw);
+    font-weight: bold;
+}
+
+.center {
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.text-medium {
+    font-size: calc(12px + 0.3vw);
+    font-weight: medium;
+    color: rgba(255, 255, 255, 0.7);
+    text-decoration: none;
+}
+
+.text-medium:hover {
+    color: rgba(255, 255, 255, 1);
+    transition: .2s linear;
+}
+
+nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: calc(25px + 1.5vw);
+    width: 85%;
+}
+
+#logo {
+    width: calc(120px + 5vw);
+}
+
+#center-nav {
+    display: flex;
+    justify-content: space-evenly;
+    width: calc(45% - 4vw);
+}
+
+#center-nav>* {
+    color: rgba(255, 255, 255, 0.7);
+    text-decoration: none;
+    font-size: calc(12px + 0.3vw);
+    font-weight: medium;
+}
+
+#center-nav>*:hover {
+    color: white;
+    transition: .2s linear;
+}
+
+#dashboard {
+    cursor: pointer;
+    text-decoration: none;
+}
+
+#dashboard>img {
+    transition: .25s linear;
+    width: calc(10px + 0.5vw);
+}
+
+#dashboard>p {
+    margin-left: calc(4px + 0.4vw);
+}
+
+#dashboard:hover>a {
+    color: rgb(255, 255, 255, 1);
+}
+
+blue {
+    color: #5A6EF8;
+}
+
+#tagline {
+    color: rgba(187, 196, 255, 0.5);
```

Comment:
everywhere things can be reused, please use scss variables. probably here for example the color should be a variable

variables should be defined at the top with well defined names

Reply:
Done 👍